frontend/src/store/bs-space/actions.ts

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
this is part of the creation page. so name it as `frontend/src/store/bs-space-create/actions.ts`